test/welcomePage.spec.js

```diff
@@ -0,0 +1,21 @@
+import { shallowMount, mount } from "@vue/test-utils"
```

Comment:
Few tips about tests organization:
1. I would create separate directories in main `test` and divide it for at least components and pages. It will be easier to maintain when there will be a lot of components and more pages.
2. You called this file "welcomePage" but I don't see such component or page. It's hard to identify at the first moment what this file is testing. It should be: 1 test file per 1component/page with the same name.